Makefile

```diff
@@ -61,7 +61,15 @@ endif
 
 DATE    := $(shell date -u '+%Y-%m-%d %H:%M UTC')
 LDFLAGS := -ldflags='-w -X "main.Version=$(VERSION)" -X "main.BuildTime=$(DATE)"'
-GOFLAGS := CGO_ENABLED=0
+GOFLAGS := -v
+
+ifeq (,$(GOFLAGS))
+	ifeq (,$(findstring CGO_ENABLED=0,$(GO_ENVS)))
```

Comment:
I guess this is for backward compatibility, right?

Reply:
Yes. If we decide to go with this approach, then I'll add a comment to it to be clear.